README.md

```diff
@@ -73,6 +69,24 @@ To opt-in to logging, the customer must:
 2. install the `elasticsearch` operator
 3. create `ClusterLogging` CR in `openshift-logging`
 
+## EFS Enablement via CSI
+
+[`efs-csi`](deploy/efs-csi) enables AWS EFS via the CSI driver. Customer
+opts in by opening a SNow ticket, whereupon SRE must add the appropriate
```

Comment:
I think we format it as `SNOW` instead of `SNow`...

Reply:
I just expanded it to ServiceNow